### PR TITLE
replace bunch of get(x).is_some() by contains(x)

### PR DESCRIPTION
**Pull request by tardyp**
was doing some cleanup algorithms on my a2l files, and found out that there is more or less same algorithm already in the public a2lfile api, but implemented with get(x).is_some()

It is actually implemented the same at low level, but this is a arguably  more idiomatic

### CODE REVIEW

File: a2lmacros/src/codegenerator/writer.rs [hunk 1]
```diff
@@ -70,7 +70,7 @@ fn generate_block_writer_generic(typename: &str, structitems: &[DataItem]) -> To
 
     quote! {
         impl #typeident {
-            pub(crate) fn stringify(&self, indent: usize) -> String {
+            pub fn stringify(&self, indent: usize) -> String {
```
Comment (DanielT) [inline a2lmacros/src/codegenerator/writer.rs]: If the stringify function is useful to you, then it could definitely be made public.
That should be a separate commit though, not part of the cleanup.

**Comment by DanielT** [comment]
Aside fom the stringify change this is a nice cleanup. If you push an update, I will merge the change.

File: a2lmacros/src/codegenerator/writer.rs (hunk 1 above)
Comment (tardyp) [reply a2lmacros/src/codegenerator/writer.rs]: oups! this is some debug leftovers. sorry!
